force-app/main/default/classes/TagAnythingBrowserControllerTest.cls

```diff
@@ -0,0 +1,35 @@
+@isTest
+public with sharing class TagAnythingBrowserControllerTest {
+
+    @isTest
+    public static void requireNewTag() {
```

Comment:
Between `Test.startTest()` and `Test.stopTest()` there should only be the thing that we want test. In this case the call to `TagAnythingBrowserController.requireTags(some_string);`